common/src/main/java/com/wynntils/features/ui/WynntilsContentBookFeature.java

```diff
@@ -88,7 +90,7 @@ public class WynntilsContentBookFeature extends Feature {
     public final Config<Boolean> replaceWynncraftContentBook = new Config<>(true);
 
     @RegisterConfig
-    public final Config<Boolean> contentBookShouldOpenWynntilsMenu = new Config<>(false);
+    public final Config<InitialPage> initialPage = new Config<>(InitialPage.USER_PROFILE);
```

Comment:
But why did you change the default? By default, we used to open the quest book, now we open the user profile.

I don't mind not having the upfixer, but let's open the quest book by default.

Reply:
Okay, I see, it was confusing. Well, it is confusing either way..